src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/mds-summary.pipe.ts

```diff
@@ -42,22 +42,49 @@ export class MdsSummaryPipe implements PipeTransform {
       contentLine1 = `${active} ${this.i18n('active')}`;
       contentLine2 = `${standbys + standbyReplay} ${this.i18n('standby')}`;
     }
-
+    let standbyHoverText = '';
+    let standbyflg = false;
+    for (const i in value.standbys) {
+      if (standbyflg) {
+        standbyHoverText = standbyHoverText + ', ' +  value.standbys[i].name;
+      } else {
+        standbyHoverText = standbyHoverText + value.standbys[i].name;
+        standbyflg = true;
+      }
+    }
+    const fsLength = (value.filesystems) ? value.filesystems.length : 0;
+    const infoObject = fsLength > 0 ? value.filesystems[0].mdsmap.info : {};
+    let activeHoverText;
+    Object.keys(infoObject).forEach(key => {
+      activeHoverText = infoObject[key].name;
+    });
+    const standbyTitleText = !standbyHoverText
+      ? ''
+      : `${this.i18n('standby daemons')}: ${standbyHoverText}`;
+    let activeTitleText = _.isUndefined(activeHoverText)
+      ? ''
+      : `${this.i18n('active daemon')}: ${activeHoverText}`;
+    if (!active && fsLength > 0) { // There is always one standbyreplay to replace active daemon, if active one is down
```

Comment:
Could you put the long comment to it's own line above the clause?
BTW in the line beneath the indent is wrong.

Reply:
Should I remove the comment instead?

Reply:
As it's a longer comment it makes sense to put it in the line above your `if`-statement, for better readability.

Reply:
Okay